src/js/components/InspectorSidebar.jsx

```diff
@@ -29,9 +29,10 @@ var Inspector = React.createClass({
   render: function() {
     var props = this.props,
         // props.id existence check handles the initial application render
-        primitive = props.id ? lookup(props.id) : {},
-        from = lookup(primitive.from),
-        ctor = primitive.constructor.name,
+        primitive = props.id ? lookup(props.id) : {};
+
+    var from = primitive ? lookup(primitive.from) : '',
+        ctor = primitive ? primitive.constructor.name : '',
```

Comment:
Might be good to add a comment indicating that these checks are only relevant if props.id is not found in the primitive store, b/c once the store evolves further the conditional logic here may be obviated